app/src/main/java/com/cobresun/brun/pantsorshorts/repositories/impl/SharedPrefsUserDataRepository.java

```diff
@@ -3,6 +3,7 @@
 import android.content.Context;
 import android.content.SharedPreferences;
 
+import com.cobresun.brun.pantsorshorts.presenter.MainActivityPresenter;
```

Comment:
Should be `import static com.cobresun.brun.pantsorshorts.presenter.MainActivityPresenter.HOURS_SPENT_OUT;`

Reply:
Import no longer needed